### DISCUSSION TITLE
Use of SMAP L3_E Data in LIS

### DISCUSSION THREAD

**Discussion by dfrancis827** [NASA-LIS/LISF#1126] [General]
I have successfully ran through the testcases in tutorial, including the assimilation of open-loop and SMAP Level 3 soil moisture estimates. However, I am attempting to assimilate SMAP Level 3 Enhanced (9 km) soil moisture using the NOAH 3.6 LSM and following the same steps described in the tutorial. I am able to do the following: (1) Process LSM parameters using LDT, (2) Conduct an Open Loop run over my domain (lower left lat/long: 38.831, -84.781 upper right lat/long: 39.147, -84.355, dx/dy: 0.01), (3) Create an ensemble restart file, (4) Construct CDF from OL run, and (5) Construct CDF from SMAP L3_E data. However. when I attempt to run assimilation between OL and SMAP L3_E data, I do not get the "DAOBS" folder as an output in the SMAPDA_OUTPUT directory. Because of this, I get multiple errors when running LVT on the SMAP assimilation saying missing files. I have attached my config file for the LIS assimilation run.

Based on comparing my attempt at running LIS over a smaller domain and smaller resolution, is it possible that I am attempting to use too small of a domain and/or too fine a resolution to be able to successfully assimilate L3_E data?

[lis.config_noah36_smapda_L3E.txt](https://github.com/NASA-LIS/LISF/files/8913334/lis.config_noah36_smapda_L3E.txt)

**Comment by bmcandr** [comment]
Hi @dfrancis827, can you also share your `lislog.0000` file?

**Reply by dfrancis827** [reply]
Certainly, I have attached my lislog_smapda.0000 file. I did have to run LIS again to create this file due to myself running it with 36 km SMAP L3 data after posting this discussion. After rerunning LIS with 9 km SMAP L3_E data, I did get the DAOBS folder, but with less OBS files than were shown in the tutorial. However, when I run the LVT for SMAP DA obs and OL DA and then try and plot point data at a given point, I do not get any SMAP observations plotted. I get the OL plot and the DA plot, but no SMAP observations. I have also attached the LVT config files for both OL and SMAP runs, my location file, and my plot file. 

[lislog_smapda.0000.txt](https://github.com/NASA-LIS/LISF/files/8921505/lislog_smapda.0000.txt)
[lvt.config.ol.da.txt](https://github.com/NASA-LIS/LISF/files/8921506/lvt.config.ol.da.txt)
[lvt.config.smapDAobs.txt](https://github.com/NASA-LIS/LISF/files/8921508/lvt.config.smapDAobs.txt)
[lvtlog.smapDAobs.0000.txt](https://github.com/NASA-LIS/LISF/files/8921509/lvtlog.smapDAobs.0000.txt)
[lvtlog_ol_da.0000.txt](https://github.com/NASA-LIS/LISF/files/8921510/lvtlog_ol_da.0000.txt)
[plot_noah36_smapda_DoeRun.txt](https://github.com/NASA-LIS/LISF/files/8921511/plot_noah36_smapda_DoeRun.txt)
[TS_LOCATIONS.TXT](https://github.com/NASA-LIS/LISF/files/8921513/TS_LOCATIONS.TXT)

**Comment by guesswhoam** [comment]
Hi @dfrancis827, I’m facing the same issue with SMAPDA_OUTPUT/DAOBS/ not generating files in Step 6 of the LISF Public Testcase. Have you resolved this problem? Any insights would be appreciated!

Thanks,

**Reply by dfrancis827** [reply]
It’s been a couple years since I worked with this, but I believe I fixed the issue. If I recall correctly, there were quality control flags within the code that were flagging the L3 Enhanced moisture I was attempting to assimilate. If I recall correctly, the main control flag that was problematic was that of vegetative water content (VWC). I was working with data over heavily vegetated areas, so I had to go in and adjust the control flag to allow for L3 enhanced from areas with higher VWC to be included in the assimilation. Once I did that, I was able to get the DAOBS folder and continue my work. If this sounds like the issue you’re having, email me at danny.francis@uky.edu and I’ll help with where the control flags are in the code.

**Reply by guesswhoam** [reply]
Hi @dfrancis827, I emailed you about this issue at [danny.francis@uky.edu](mailto:danny.francis@uky.edu) from my email address ([huwk7@mail3.sysu.edu.cn](mailto:huwk7@mail3.sysu.edu.cn)) a while ago but haven't heard back. I'm wondering if perhaps my email went to your spam folder? If you could check your spam/junk folder or let me know if there's a better way to reach you, I'd greatly appreciate it. Thanks!
